src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -1337,7 +1349,7 @@ namespace HeatBalanceManager {
         }
 
         static constexpr auto Format_732(
-            "! <Zone Air Mass Flow Balance Simulation>, Enforce Mass Balance, Adjust Zone Mixing, Adjust Zone Infiltration "
+            "! <Zone Air Mass Flow Balance Simulation>, Enforce Mass Balance, Adjust Zone Mixing and Return {{AdjustMixingOnly | AdjustReturnOnly | AdjustMixingThenReturn | AdjustReturnThenMixing | None}}, Adjust Zone Infiltration "
```

Comment:
This update replaces the single "Adjust Zone Mixing" with "Adjust Zone Mixing and Return" and added the five choice key options. This change impacts the EIO output file and the tabular column headings.